src/s11/handlers/delete_bearer_resp_handler.c

```diff
@@ -64,27 +63,33 @@ db_resp_processing(struct DB_RESP_Q_msg *db_resp_msg)
     
 	msgData.cause.causeValue = db_resp_msg->cause;
 
-	if(db_resp_msg->bearer_ctxt_db_resp_list.bearers_count > 0)
-	{
-	    msgData.bearerContextsCount = db_resp_msg->bearer_ctxt_db_resp_list.bearers_count;
-
-	    for(int i=0; i < msgData.bearerContextsCount; i++)
-	    {
-	        msgData.bearerContexts[i].epsBearerId.epsBearerId = db_resp_msg->bearer_ctxt_db_resp_list.bearer_ctxt[i].eps_bearer_id;
-
-		msgData.bearerContexts[i].cause.causeValue = db_resp_msg->bearer_ctxt_db_resp_list.bearer_ctxt[i].cause.data;
-
-		if(db_resp_msg->bearer_ctxt_db_resp_list.bearer_ctxt[i].pco_from_ue_opt.pco_length > 0)
-		{
-		    msgData.bearerContexts[i].protocolConfigurationOptionsIePresent = true;
-		    msgData.bearerContexts[i].protocolConfigurationOptions.pcoValue.count =
-				    db_resp_msg->bearer_ctxt_db_resp_list.bearer_ctxt[i].pco_from_ue_opt.pco_length;
-		    memcpy(msgData.bearerContexts[i].protocolConfigurationOptions.pcoValue.values, 
-				    db_resp_msg->bearer_ctxt_db_resp_list.bearer_ctxt[i].pco_from_ue_opt.pco_options,
-				    db_resp_msg->bearer_ctxt_db_resp_list.bearer_ctxt[i].pco_from_ue_opt.pco_length);
-		}
-	    }
-	}
+    if (db_resp_msg->bearer_ctxt_db_resp_list.bearers_count > 0)
```

Comment:
Need handling of rejection of delete bearer request

Reply:
Yes, we shall handle it in dedicated bearer deactivation.